custom_components/powercalc/data/signify/LLC006/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Signify",
```

Comment:
This needs to be the model/type of smart plug you used to measure the hue light.